recipes/dagster-cloud/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e  # Exit immediately if a command exits with a non-zero status.
+
+# Install pex using pip
+$PYTHON -m pip install pex<3,>=2.1.132
```

Comment:
```suggestion
$PYTHON -m pip install "pex<3,>=2.1.132"
```

This is the error, but you can't install pip stuff here as part of the recipe. If pex is not on conda-forge, create a recipe for it and put it under `recipes/`. The infra will figure out the build order.

Reply:
Thanks! I applied your suggestion.